code/drasil-data/lib/Data/Drasil/Equations/Defining/Physics.hs

```diff
@@ -13,9 +13,9 @@ import Data.Drasil.Concepts.Documentation (body, constant)
 -- * Equations
 
 weightEqn, newtonSLEqn, hsPressureEqn, speedEqn :: ExprC r => r
-newtonSLEqn               = sy QPP.mass `mulRe` sy QP.acceleration
-weightEqn                 = sy QPP.vol `mulRe` sy QPP.specWeight
-hsPressureEqn             = sy QPP.specWeight `mulRe` sy QP.height
+newtonSLEqn               = sy QPP.mass `mul` sy QP.acceleration
```

Comment:
Nice! I think that a 'next step' would be to replace  `mul` and `add` with `$*` and `$+`, respectively. You can use the definition of `$-` for inspiration on how to define these.